README.md

```diff
@@ -1,6 +1,68 @@
 # Books and Bookcases
 
-This module adds the ability to read and write books, as well as store them in bookcases.
+This module adds the ability to read and write books, as well as store them in bookcases. This module also adds the ability to edit books and their contents.
 
-Credits for images:
-- Quill - https://openclipart.org/detail/262818/vintage-feather-inkwell
\ No newline at end of file
+The Following Types of Components Can Be Made:
+* `Book`
+* `Bookcase`
+
+
+# Contribution
+
+Fork and clone this repository locally
```

Comment:
we tend not to clone the module repository directly into the terasology workspace. there is some setup that gradle and groovyw will do when you grab a module. there is some documentation in the wiki that explains this already: https://github.com/MovingBlocks/Terasology/wiki/Developing-Modules

Reply:
This contradicts the examples you provided. [Ex1](https://github.com/Terasology/Potions) [Ex2](https://github.com/Terasology/Cooking) .

Reply:
Oh that's a good catch. Probably those were introduced, not caught at the time, then have spread around as poor examples. Feel free to fix that up for any other modules you notice it for. Not really a separate task but if you do it and highlight that to us that counts for intangible Cool Points to possibly get put to use on other stuff in the future :-)

Reply:
That sounds great. I'll do it when I get some time!